contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -173,11 +177,13 @@ contract KeeperRegistryDev is
     uint16 gasCeilingMultiplier,
     uint256 fallbackGasPrice,
     uint256 fallbackLinkPrice,
-    bool keepersMustTakeTurns
+    bool keepersMustTakeTurns,
+    uint32 maxPerformGas
   ) ConfirmedOwner(msg.sender) {
     LINK = LinkTokenInterface(link);
     LINK_ETH_FEED = AggregatorV3Interface(linkEthFeed);
     FAST_GAS_FEED = AggregatorV3Interface(fastGasFeed);
+    s_maxPerformGas = maxPerformGas;
```

Comment:
can we move this to the `Config` struct?

Reply:
as per offline discussion, moved it to config and allow it to be increased